### DISCUSSION TITLE
Eingabe von SMS Code geht nicht

### DISCUSSION THREAD

**Discussion by Uwe96** [flobz/psa_car_controller#484] [General]
Hallo,
nach Eingabe des SMS Codes bekomme ich folgende Fehlermeldung:
{'err': 'NOK:SN'}
Habe es mehrmals versucht. Der Code ist richtig eingegeben

**Comment by Uwe96** [comment]
Habe es selbst rausgefunden.
Mit Firefox ging es nicht. 
Aber mit Edge hat es geklappt :)

**Comment by miguelmaugusto** [comment]
If you have the same issue as me, it was a bcak error. Chrome filled out the first field (code sent by sms) with the email and I was entering the code in the last field. The last field is for your pin code for the mypeugeot app

**Reply by matilbianco** [reply]
Hello
I did not find where to setup PIN code in the MyPeugeot app. Can you help me?
Also I do not receive OTP code when HA plug-in says "SMS sent"
